### PR TITLE
Add support for Cocoapods

### PR DESCRIPTION
**Pull request by xmollv**
@igorcferreira I've tested this locally and seems to work (pointing to the .podspec from a Podfile). Please, take a look at it anyhow and let me know if you see any issues!

I'm assuming that we'll also have to publish this to `MWPodspecs`, right?

### CODE REVIEW

File: FancyScrollView.podspec [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    s.name                  = 'FancyScrollView'
+    s.version               = '0.1.3'
+    s.summary               = 'A SwiftUI ScrollView Designed to imitate the App Store and Apple Music ScrollViews'
+    s.description           = <<-DESC
+    A SwiftUI ScrollView Designed to imitate the App Store and Apple Music ScrollViews (with or without a Parallax Header)
+    DESC
+    s.homepage              = 'https://github.com/nerdsupremacist/FancyScrollView'
+    s.license               = { :type => 'Copyright', :file => 'LICENSE' }
+    s.author                = { 'Mathias Quintero' => 'me@quintero.io' }
+    s.source                = { :git => 'https://github.com/nerdsupremacist/FancyScrollView.git', :tag => "#{s.version}" }
```
Comment (xmollv) [inline FancyScrollView.podspec]: I've pointed everything to the original. Is this how it should be or should we be pointing to our fork?

Comment (igorcferreira) [reply FancyScrollView.podspec]: Giving that we have the fork, it is better to point to our forked code. This way, we have a true reference of the code that is being provided

**Comment by igorcferreira** [comment]
> I'm assuming that we'll also have to publish this to `MWPodspecs`, right?

Yes. We will publish this into MWPodspecs

File: FancyScrollView.podspec (hunk 1 above)
Comment (xmollv) [reply FancyScrollView.podspec]: @igorcferreira Got it! Updated the source to our fork 👍🏼

**Review by igorcferreira — APPROVED** [review]

**Pull request merged**